src/cli/config/bundler.ts

```diff
@@ -83,7 +83,10 @@ export const bundlerArgsSchema = z.object({
     "max-gas-per-bundle": z
         .string()
         .transform((val) => BigInt(val))
-        .default("5000000")
+        .default("5000000"),
+    "stateless-enabled": z
```

Comment:
Would it better to just take a list of methods that can be enabled?
And we can pass the list like:
`eth_chainId,eth_supportedEntryPoints,eth_estimateUserOperationGas,eth_getUserOperationByHash,eth_getUserOperationReceipt`

And have a separate param to enable of disable refill the wallets?

Reply:
hm, actually that's a good idea